packages/RatPack.package/RPApplication.class/instance/setRouteHandlerFor..st

```diff
@@ -0,0 +1,7 @@
+routing
+setRouteHandlerFor: aHttpRequest
```

Comment:
should RPApplication>>route: and RPApplication>>parseRequestContent: also be refactored and use RPApplication>>request ??

because request is set in RPApplication>>processRequest: ...